samples/Plugin.FilePicker.Sample.Forms/Plugin.FilePicker.Sample.Forms/Plugin.FilePicker.Sample.Forms.csproj

```diff
@@ -42,13 +42,13 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Xamarin.Forms.Core">
-      <HintPath>..\packages\Xamarin.Forms.2.5.0.121934\lib\portable-win+net45+wp80+win81+wpa81\Xamarin.Forms.Core.dll</HintPath>
+      <HintPath>..\..\packages\Xamarin.Forms.2.5.0.121934\lib\portable-win+net45+wp80+win81+wpa81\Xamarin.Forms.Core.dll</HintPath>

```

Comment:
What is going on with all the path changes here?

Reply:
It seems the sample project Plugin.FilePicker.Sample.Forms was moved a folder deeper, but the solution .sln file wasn't. The old csproj format stores direct references to the packages that are added to the project, and packages are by default restored to the "packages" folder relative to the .sln file. This change fixes the path.